_includes/components/header--basic.html

```diff
@@ -50,18 +50,20 @@
         {% endif %}
 
         {% assign _secondary = header.secondary %}
-
-
-          {% if _secondary.search %}
-          <div class="usa-search usa-search-small" role="search">
-            <form action="{{ _secondary.search.action | relative_url }}">
-                <label class="usa-sr-only" for="search-field-small">Search small</label>
-                <input id="search-field-small" type="search" name="{{ _secondary.search.query | default: 'search' }}">
-                <button type="submit">
+          {% if site.search  %}
+            <form accept-charset="UTF-8" action="https://search.usa.gov/search" id="search_form" method="get" class="usa-search usa-search-small js-search-form">
+              <div style="margin:0;padding:0;display:inline">
```

Comment:
I think this `div` can be removed since all it contains is a hidden form value.

Reply:
I wasn't sure if that hidden value was needed, this code came directly from https://search.gov/manual/code.html

Reply:
Oh yeah, looking at it now. That's strange, I guess leave it?